lib/model/rwfolder.go

```diff
@@ -1290,11 +1239,8 @@ func (f *sendReceiveFolder) copierRoutine(in <-chan copyBlocksState, pullChan ch
 
 			if !found {
 				found = f.model.finder.Iterate(folders, block.Hash, func(folder, file string, index int32) bool {
-					inFile, err := rootedJoinedPath(folderRoots[folder], file)
-					if err != nil {
-						return false
-					}
-					fd, err := os.Open(inFile)
+					fs := folderFilesystems[folder]
+					fd, err := fs.Open(file)
```

Comment:
The parameter being named `file` confused the fuck out of me when reading the diff as there is now a `file` declared at the function scope which is of a completely different type. Rename one?